### PR TITLE
Release/0.6.0

### PR DESCRIPTION
**Pull request by pondzix**

### CODE REVIEW

**Review by chuwy — APPROVED** [review]
Have really nothing to pick up on. Looks perfect!

Maybe just replace lib names to actual artifact names, like:

`cats effect` -> `cats-effect`

File: README.md [hunk 1]
```diff
@@ -82,7 +82,19 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
 [license]: http://www.apache.org/licenses/LICENSE-2.0
+
+[ci]: https://github.com/snowplow-incubator/scala-lru-map/actions?query=workflow%3ACI
+[ci-image]: https://github.com/snowplow-incubator/scala-lru-map/workflows/CI/badge.svg
+
+[releases]: https://maven-badges.herokuapp.com/maven-central/com.snowplowanalytics/scala-lru-map_2.13
+[release-image]: https://img.shields.io/maven-central/v/com.snowplowanalytics/scala-lru-map_2.13.svg
+
+[coveralls]: https://coveralls.io/github/snowplow/scala-lru-map?branch=master
+[coveralls-image]: https://coveralls.io/repos/github/snowplow/scala-lru-map/badge.svg?branch=master
```
Comment (paulboocock) [inline README.md]: These badges are slightly wrong (noticed whilst trying to figure out why Coveralls is failing). Should be in `snowplow-incubator`.

```
[![Coverage Status](https://coveralls.io/repos/github/snowplow-incubator/scala-lru-map/badge.svg?branch=master)](https://coveralls.io/github/snowplow-incubator/scala-lru-map?branch=master)
```

**Comment by paulboocock** [comment]
I don't have much to add to this, besides trying to help with the secrets and coveralls. Things look good from that point of view 👍

**Review by chuwy — APPROVED** [review]
https://github.com/snowplow-incubator/scala-lru-map/pull/47/commits/309d1d577936578bec36afc7af62aa68d09cb272 also looks awesome - I honestly never fully understood how the previous implementation was working, now it's much clearer

**Pull request merged**